### ISSUE TITLE
Build failing: package android.support.v4.content does not exist

### ISSUE THREAD

**Issue by renancunha** [react-native-community/react-native-image-picker#1092]
# Bug

When I try to run my react-native application on android I'm facing some issues related to package ``android.support``. I think that could be related to the fact that I'm building to android x.

This library currently supports Android X?

## Environment info

  React Native Environment Info:
    System:
      OS: Windows 10
      CPU: (8) x64 Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz
      Memory: 6.90 GB / 15.65 GB
    Binaries:
      Yarn: 1.13.0 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
      npm: 6.4.1 - C:\Program Files\nodejs\npm.CMD
    IDEs:
      Android Studio: Version  3.2.0.0 AI-181.5540.7.32.5056338

React native info output:

```bash
node_modules\react-native-image-picker\android\src\main\java\com\imagepicker\FileProvider.java:3: error: package android.support.v4.content does not exist
public class FileProvider extends android.support.v4.content.FileProvider {
                                                            ^
node_modules\react-native-image-picker\android\src\main\java\com\imagepicker\ImagePickerModule.java:15: error: package android.support.annotation does not exist
import android.support.annotation.NonNull;
                                 ^
node_modules\react-native-image-picker\android\src\main\java\com\imagepicker\ImagePickerModule.java:16: error: package android.support.annotation does not exist
import android.support.annotation.Nullable;
                                 ^
node_modules\react-native-image-picker\android\src\main\java\com\imagepicker\ImagePickerModule.java:17: error: package android.support.annotation does not exist
import android.support.annotation.StyleRes;
                                 ^
node_modules\react-native-image-picker\android\src\main\java\com\imagepicker\ImagePickerModule.java:18: error: cannot find symbol
import android.support.v4.app.ActivityCompat;
                             ^
  symbol:   class ActivityCompat
  location: package android.support.v4.app
node_modules\react-native-image-picker\android\src\main\java\com\imagepicker\ImagePickerModule.java:19: error: package android.support.v7.app does not exist
```

Library version: Latest (0.28.1)

## Steps To Reproduce

Just add the library to the project, link it and then run ``react-native run-android``.

Describe what you expected to happen:

1. Build and run the application normally

**Comment by Gaafar** [comment]
same error here, only started after I added and linked firebase `@react-native-firebase/app@alpha`

**Comment by Gaafar** [comment]
Running `Refactor` Menu > `Migrate to AndroidX` fixed the error for me

**Comment by artyom-developer** [comment]
@Gaafar  but i have other error

this is code not support:

final AlertDialog dialog = UI.chooseDialog(this, options, new UI.OnAction()
    {
      @Override
      public void onTakePhoto(@NonNull final ImagePickerModule module)
      {
        if (module == null)
        {
          return;
        }
        module.launchCamera();
      }

      @Override
      public void onUseLibrary(@NonNull final ImagePickerModule module)
      {
        if (module == null)
        {
          return;
        }
        module.launchImageLibrary();
      }

      @Override
      public void onCancel(@NonNull final ImagePickerModule module)
      {
        if (module == null)
        {
          return;
        }
        module.doOnCancel();
      }

      @Override
      public void onCustomButton(@NonNull final ImagePickerModule module,
                                 @NonNull final String action)
      {
        if (module == null)
        {
          return;
        }
        module.invokeCustomButton(action);
      }
    });

**Comment by MonkeyMoneyFrance** [comment]
check react-native-camera version is below 2.10 (androidX implementation)

**Comment by VitorBrangioni** [comment]
Same issue. Any solution?

**Comment by Johan-dutoit** [comment]
Resolved in the latest version [v1.0.1](https://github.com/react-native-community/react-native-image-picker/releases/tag/v1.0.1).

**Comment by HarshitMadhav** [comment]
@Johan-dutoit I am getting the same error and I am using 0.28.0 version of react native image picker and I am not using androidx
